app/src/main/java/com/example/insights/BookFragment.kt

```diff
@@ -24,17 +24,17 @@ class BookFragment : Fragment() {
 
         var recyclerView: RecyclerView = root.findViewById<RecyclerView>(R.id.recyclerview)
         recyclerView.layoutManager = LinearLayoutManager(context)
-        //val itemAdapter= ItemAdapter(this,bookvalues)
-        //recyclerView.adapter=(itemAdapter)
+        //val BookRecylerViewAdapter= BookRecylerViewAdapter(this,bookvalues)
+        //recyclerView.adapter=(BookRecylerViewAdapter)
         val db = FirebaseFirestore.getInstance()
         db.collection("Books").addSnapshotListener { value, error ->
             if (error == null) {
                 if (value != null) {
                     for (change in value.documentChanges) {
                         if (change.type == DocumentChange.Type.ADDED) {
                             bookvalues.add(change.document.data as HashMap<String, String>)
-                            val Itemadapter = ItemAdapter(this, bookvalues)
-                            recyclerView.adapter = Itemadapter
+                            val BookRecylerViewAdapter = BookRecylerViewAdapter(this, bookvalues)
```

Comment:
NIT fix, make the name of this value book_recycler_view_adapter instead of BookRecyclerViewAdapter